src/main/java/com/cinema/model/dto/TicketResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package com.cinema.model.dto;
+
+public class TicketResponseDto {
+    private Long id;
+    private MovieSessionResponseDto session;
```

Comment:
the `tickets` list in Cart/OrderResponseDto becomes more and more complex

let's have some really simple fields in this TicketResponseDto
or return `List<Long> ticketIds` in Cart/OrderResponseDto